src/server/utils/getIP.js

```diff
@@ -0,0 +1,12 @@
+import { networkInterfaces } from 'os';
+
+export const getIp = () => {
+  const interfaces = networkInterfaces();
+
+  const ipAddress = Object.keys(interfaces)
```

Comment:
```suggestion
  const ipAddresses = Object.keys(interfaces)
```

just because it's an array